_data/simple-icons.json

```diff
@@ -2819,6 +2819,16 @@
             "hex": "EF4223",
             "source": "https://www.codeigniter.com/help/legal"
         },
+        {
+            "title": "Codeium",
+            "hex": "09B6A2",
+            "source": "https://exafunction.github.io/simple-logo.svg",
```

Comment:
@khou22 - still not happy with this being a github.io link, could you please provide a public GitHub repo link for this, so that we can monitor any changes against it?

Reply:
@adamrusted Sure thing. Just uploaded the asset to one of our public GitHub repos: https://github.com/Exafunction/codeium.vim/blob/main/codeium-simple-logo.svg